native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/PasscodeActivityTest.java

```diff
@@ -455,14 +455,24 @@ public void testForgotPasscodeLink() throws Throwable {
 
         // Clicking on 'Cancel' should take us back to the passcode screen.
         final AlertDialog logoutDialog = passcodeActivityTestRule.getActivity().getLogoutAlertDialog();
-        logoutDialog.getButton(AlertDialog.BUTTON_NEGATIVE).performClick();
+        passcodeActivity.runOnUiThread(new Runnable() {
```

Comment:
Please let me know if there is a cleaner way to do this.

Reply:
`Espresso` was supposed to provide a way to not do this, but I don't think it handles overlays like `AlertDialog` well. This is fine.